witchcraft-server/src/minidump/mod.rs

```diff
@@ -69,6 +64,34 @@ pub async fn init() -> Result<(), Error> {
     Ok(())
 }
 
+#[cfg(target_os = "linux")]
+fn handle_restricted_ptrace(child: u32) -> Result<(), Error> {
+    let ptrace_scope = match fs::read_to_string("/proc/sys/kernel/yama/ptrace_scope") {
```

Comment:
This file won't exist when yama isn't running.

Reply:
This is probably worth noting as a comment? And ideally linking to some docs on this

Reply:
Done!